src/_guides/language/language-tour_migrated.md

```diff
@@ -4111,8 +4129,8 @@ When you need to lazily produce a sequence of values,
 consider using a _generator function_.
 Dart has built-in support for two kinds of generator functions:
 
-* **Synchronous** generator: Returns an **[Iterable]** object.
-* **Asynchronous** generator: Returns a **[Stream]** object.
+* **Synchronous** generator: Returns an **[`Iterable`]** object.
+* **Asynchronous** generator: Returns a **[`Stream`]** object.
```

Comment:
Maybe delete the `**` bold since it probably doesn't play nice with the code font?